### PR TITLE
Documentation Update / Refactor

### PR DESCRIPTION
**Pull request by alanscandrett**
We have updated a series of files that were modified through the sprint, and added a series of diagrams, alongside this we added a table of contents.

Authors: Stella (Live EO), Alan (EUSI)

### CODE REVIEW

File: .markdownlint.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (m-mohr) [inline .markdownlint.json:1]: Should be removed.

File: order/README.md [hunk 2]
```diff
@@ -61,13 +59,13 @@ If the `GET /orders/{orderId}/status` endpoint is implemented, there must be a l
...
-| timestamp  | datetime | **REQUIRED.** ISO 8601 timestamp for the order status |
-| status_code | string | **REQUIRED.** Enumerated status code |
-| reason_code | string | Enumerated reason code for why the status was set |
-| reason_text | string | Textual description for why the status was set |
-| links | \[Link Object\] | **REQUIRED.** list of references to documents, such as delivered asset, processing log, delivery manifest, etc. |
+| timestamp  | datetime | ISO 8601 timestamp for the order status (required) |
+| status_code | string | Enumerated status code (required) |
+| reason_code | string | Enumerated reason code for why the status was set (optional) |
+| reason_text | string | Textual description for why the status was set (optional) |
+| links | [Link] | list of references to documents, such as delivered asset, processing log, delivery manifest, etc. (required, may be empty) |
```
Comment (m-mohr) [inline order/README.md:66]: This overrides existing changes, probably a failed merge

File: order/README.md [hunk 3]
```diff
@@ -96,10 +94,10 @@ Providers may support these statuses.
 
 #### Extension status codes
 
-Providers may support additional statuses through extensions. For example:
+Providers may support additional statuses through extensions.  For example:
 
 * tasked (indicates tasking commands have been issued to the satellite/constellation)
-* user_cancelled (indicates that the user cancelled the request)
+* user_cancelled (indicates that )
```
Comment (m-mohr) [inline order/README.md:100]: Same here

**Review by m-mohr — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -73,17 +65,14 @@ The following table describes the service resources available in a STAT API impl
...
-| `GET /orderds/{orderId}`               | Core          | -                                                            | [Order Object](./order/README.md#order-pobject)              |                                                              |
-| `POST /orders`                         | Core          | [Order Request](./order/README.md#order-request)             | [Order Object](./order/README.md#order-pobject)              | Order a capture with a particular set of constraints         |
-| `POST /opportunities`                  | Opportunities | [Opportunity Request](./opportunity/README.md#opportunity-request) | [Opportunities Collection](./opportunity/README.md#opportunities-collection) | Explore the opportunities available for a particular set of constraints |
+| Endpoint                    | Specified in  | Accepts                                                      | Returns                                                      | Description                                                  |
```
Comment (m-mohr) [inline README.md:68]: Failed merge? Removes previous changes.

File: core/examples/landingpage.json [hunk 5]
```diff
@@ -12,34 +12,17 @@
...
-            "type": "application/json",
-            "href": "https://stat-api.example.com/orders",
-            "title": "List of existing orders"
-        },
-        {
-            "rel": "products",
-            "type": "application/json",
-            "href": "https://stat-api.example.com/products",
-            "title": "List of available products"
+            "href": "https://stat-api.example.com/conformance"
```
Comment (m-mohr) [inline core/examples/landingpage.json:15]: Same here

File: examples/OpportunityCollectionV2.json [hunk 6]
```diff
@@ -16,7 +16,7 @@
             },
             "links": [
                 {
-                    "rel": "create-order",
+                    "rel": "order",
```
Comment (m-mohr) [inline examples/OpportunityCollectionV2.json:19]: Invalid

File: examples/OpportunityRequestV2.json [hunk 7]
```diff
@@ -13,7 +13,7 @@
     ]
   },
   "datetime": "2020-01-01T00:00:00Z/2020-01-02T00:00:00Z",
-  "filter": {
```
Comment (m-mohr) [inline examples/OpportunityRequestV2.json:16]: It's filter, not filters

File: .markdownlint.json (hunk 1 above)
Comment (alanscandrett) [reply .markdownlint.json]: Commit to remove this file has been pushedn now

**Comment by alanscandrett** [comment]
@m-mohr We made an attempt to fix the failed merge, would it be possible to have you take another look at this?

File: opportunity/README.md [hunk 8]
```diff
@@ -11,9 +11,9 @@ for POST /opportunities
 | Field Name | Type                                                                       | Description |
 | ---------- | -------------------------------------------------------------------------- | ----------- |
 | datetime       | string                                                                     | **REQUIRED.** Datetime field is a [ISO8601 Time Interval](https://en.wikipedia.org/wiki/ISO_8601#Time_intervals) |
-| product_id         | string                                                                     | **REQUIRED.** Product identifier. The ID should be unique and is a reference to the constraints which can be used in the constraints field. |
+| product_id         | string                                                                     | **REQUIRED.** Product identifier. The ID should be unique and is a reference to the [parameters](https://github.com/Element84/stat-api-spec/blob/main/product/README.md#parameters) which can be used in the [parameters](https://github.com/Element84/stat-api-spec/blob/main/product/README.md#parameters) field. |
```
Comment (m-mohr) [inline opportunity/README.md:14]: I don't quite get that sentence. Not your fault, but what?

File: product/README.md [hunk 9]
```diff
@@ -71,9 +85,3 @@ It is recommended to use [JSON Schema draft-07](https://json-schema.org/specific
 For an introduction to JSON Schema, see
 [Learn JSON Schema](https://json-schema.org/learn/getting-started-step-by-step).
 
-#### Parameters Best Practices
```
Comment (m-mohr) [inline product/README.md:74]: ?

File: stat.yaml [hunk 10]
```diff
@@ -195,15 +195,35 @@ paths:
         content:
           application/json:
             schema:
-              $ref: "#/components/schemas/order_request"
```
Comment (m-mohr) [inline stat.yaml:198]: Intentional?

**Review by ZheTioS — APPROVED** [review]
LGTM!

**Comment by m-mohr** [comment]
Closing in favor of #193

**Pull request closed without merging**